README.md

```diff
@@ -121,6 +121,18 @@ You can find a list of libraries and integrations with data stores built on top
 
 If you have a project you'd like to include in this list, either open a PR or let us know in [the gitter channel](https://gitter.im/functional-streams-for-scala/fs2) and we'll add a link to it.
 
+### Adopters ### 
+
+Here's a (non-exhaustive) list of companies that use FS2 in production. Don't see yours? You can [add it in a PR!](https://github.com/functional-streams-for-scala/fs2/edit/main/README.md)
+
+* [Chatroutelette](https://v2.chatroulette.com/)
```

Comment:
Typo

Reply:
whoops fixed.

Reply:
@mpilquist ready to merge when you are :)